modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectDefinitionLocalServiceTest.java

```diff
@@ -3106,6 +3275,25 @@ private void _assertModelResourceNames(List<String> objectDefinitionNames)
 		}
 	}
 
+	private void _assertObjectDefinitionSetting(
+		String expectedName, String expectedValue,
+		List<ObjectDefinitionSetting> objectDefinitionSettings) {
+
+		String actualValue = null;
+
+		for (ObjectDefinitionSetting objectDefinitionSetting :
+				objectDefinitionSettings) {
+
+			if (expectedName.equals(objectDefinitionSetting.getName())) {
+				actualValue = objectDefinitionSetting.getValue();
+
+				break;
+			}
+		}
+
+		Assert.assertEquals(expectedValue, actualValue);
+	}
```

Comment:
I saw that we have this method validating `ObjectFieldSettings` in `ObjectFieldLocalServiceTest.java`. Can we do similar to `ObjectDefinitionSetting`?
```
private void _assertObjectFieldSettingsValues(
	long objectFieldId, Map<String, String> objectFieldSettingsValues)
throws Exception {

      for (Map.Entry<String, String> entry :
		      objectFieldSettingsValues.entrySet()) {
      
	      ObjectFieldSetting objectFieldSetting =
		      _objectFieldSettingLocalService.fetchObjectFieldSetting(
			      objectFieldId, entry.getKey());
      
	      Assert.assertEquals(
		      entry.getValue(), objectFieldSetting.getValue());
      }
}
```